RoundsWithFriends/RoundsWithFriends.cs

```diff
@@ -197,6 +233,30 @@ public void Start()
 
                 PhotonPeer.RegisterType(typeof(DebugOptions), 77, DebugOptions.Serialize, DebugOptions.Deserialize);
             }
+            PhotonPeer.RegisterType(typeof(LobbyCharacter), 78, LobbyCharacter.Serialize, LobbyCharacter.Deserialize);
+
+            // add beta text
+            BetaTextHandler.AddBetaText(true);
```

Comment:
Remove beta stuff before merge (also the beta manifest stuff). Or alternatively hide it based on some configuration.

Reply:
Added `Beta` configuration